cmd/root.go

```diff
@@ -115,7 +115,8 @@ func rootCmdPersistentFlagSet() *pflag.FlagSet {
 func init() {
 	confDir, err := configDir()
 	if err != nil {
-		panic(err)
+		logrus.Warn("could not get config directory")
```

Comment:
I would propose we add some hint that HOME wasn't set ... but needs to be specific for each OS ? I think

Reply:
Maybe just output the error message?

Reply:
`WARN[0000] could not get config directory                error="$HOME is not defined"`

Seems better